README.adoc

```diff
@@ -50,42 +46,42 @@ This is a *highly opinionated* guide. It is not intended to provide you with a c
 
 == 1. Configuring Your Project to Use TypeScript
 
-=== Installing the TypeScript Compiler
-
-TypeScript is a language designed for developers, meaning it must be compiled into plain JavaScript before Foundry VTT can execute it.
+=== Initializing Your Project with `npm`
 
-To get started, you'll need to install the TypeScript compiler.
-The easiest way to do this is via link:https://nodejs.org/en/download/package-manager[Node.js] and its package manager, `npm`. If you don’t already have Node.js installed, download and install it from the provided link. Then, open your terminal and run the following command:
+Ensure your project is properly set up for Node.js and `npm` (i.e. it has a `package.json` file).
+You can initialize a new `npm` project by running the following terminal command from the root directory of your project:
 
 [source,console]
 ----
-npm install -g typescript
+npm init
 ----
 
-To confirm the installation, you can check the compiler version by running:
+This command will guide you through creating a `package.json` file. You can accept the default values or customize them as needed for your project.
+
+=== Installing TypeScript
+
+TypeScript is a language that compiles into JavaScript. This means before Foundry VTT can execute it must be compiled into plain JavaScript.
+
+To get started, you'll need to install the TypeScript compiler.
+The easiest way to do this is via link:https://nodejs.org/en/download/package-manager[Node.js] and its package manager, `npm`. If you don't already have Node.js installed, download and install it from the provided link. Then, open your terminal and run the following command:
 
 [source,console]
 ----
-tsc --version
+npm add --include=dev typescript
```

Comment:
Telling people to install TypeScript can useful _if they know what they're doing_ and want to compile code on the fly but since I figured this guide is trying to target the lowest common denominator for projects we should recommend adding it to their project.